components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/http/management/HTTPEndpointManager.java

```diff
@@ -427,4 +430,8 @@ public InternalAPIDispatcher getInternalAPIDispatcher() {
     public int getInternalInboundPort() {
         return internalInboundPort;
     }
+
+    public boolean isAnyInternalApiEnabled() {
+        return internalAPIList != null && internalAPIList.size() > 0;
```

Comment:
Once the if condition in line 73 is removed, we can remove the check 'internalAPIList != null' since 'ConfigurationLoader.loadInternalAPIs' will never return null